site/connections.md

```diff
@@ -0,0 +1,200 @@
+connections also increases node's memory consumption.
+
+#### Relevant Metrics
+
+[Management UI](/management.html) provides a chart of the total number of connections opened cluster-wide:
+
+<img class="screenshot" src="img/monitoring/connections/mgmt-ui-global-connection-count.png" alt="Global connection count in management UI" title="Global connection count in management UI" />
+
+A connection leak on monitoring charts can be identified as an monotonically growing number of client connections.
+
+It is also possible to see how many file handles and sockets a specific node has, which can be useful
+in determining connection leaks as well. The following chart demonstrates a very stable number of sockets
+open on a node:
+
+<img class="screenshot" src="img/monitoring/connections/mgmt-ui-node-socket-count.png" alt="Node file handle and socket count in management UI" title="Node file handle and socket count in management UI" />
+
+This chart demonstrates a monotonically growing number of connections after a drop:
+
+<img class="screenshot" src="img/monitoring/connections/mgmt-ui-node-socket-count-growth.png" alt="Node file handle and socket count growth in management UI" title="Node file handle and socket count growth in management UI" />
+
+If the number of sockets used by a node keeps growing and growing this may be an indication
+of a connection leak in one of the applications.
+
+Some client libraries, [such has the Java client](/api-guide.html#metrics), expose metrics including the number of currently
+opened connections. Charting and monitoring application metrics around connections is the best way
+to identify what app leaks connections or uses them in a suboptimal way.
+
+In many applications that use long-lived connections and do not leak them the number of connections
+grows on application start and then moderates (stays mostly stable with little fluctuation).
+
+[Management UI](/management.html) provides a chart of connection churn rate as of [RabbitMQ 3.7.9](/changelog.html).
+Below is a chart that demonstrates a fairly low connection churn with a comparable number of connections open and closed
+in the given period of time:
+
+<img class="screenshot" src="img/monitoring/connections/mgmt-ui-node-connection-churn.png" alt="Node connection churn in management UI" title="Node connection churn in management UI" />
+
+
+### High Connection Churn
+
+A system is said to have high connection churn when its rate of newly opened connections is consistently high and
+its rate of closed connection is consistently high. This usually means that an application
+uses short-lived connections. While with some workloads this is a natural state of the system,
+long-lived connections should be used instead when possible.
+
+[Management UI](/management.html) provides a chart of connection churn rate as of [RabbitMQ 3.7.9](/changelog.html).
```

Comment:
This paragraph and the image are duplicated from the previous section.

Reply:
New connection rate and connection churn are based on the same metric(s). I did some editing to make it clearer why this chart is relevant for detecting connection leaks.